### PR TITLE
(SIMP-3163) Updates for EL 6.9 and release_mappers

### PR DESCRIPTION
**Pull request by nick-markowski**
- Updated packages.yaml and ISO prune list
- Updated release mapper (NOTE RHEL 6.9 Not updated)
- Updated gitignore to include src/assets/environment, it is
  managed by the simp-environment-skeleton project

SIMP-3170 #close
SIMP-3171 #close

### CODE REVIEW

**Comment by nick-markowski** [comment]
If you want to test this, it is best to remove all refs to ELG from Puppetfile.* and src/build/simp.spec.  ELG versions will be updated as they are merged.

File: Puppetfile.tracking [hunk 1]
```diff
@@ -26,7 +26,7 @@ mod 'rubygem-simp_cli',
 moduledir 'src/puppet/modules'
 
 mod 'bfraser-grafana',
-  :git => 'https://github.com/simp/puppet-grafana',
```
Comment (jeefberkey) [inline Puppetfile.tracking]: This should still be our fork. If master needs to be updated that's another ticket.

**Comment by nick-markowski** [comment]
Don't merge yet! Making some final release mapper updates

**Comment by nick-markowski** [comment]
When merging, ref SIMP-3170, SIMP-3171, SIMP-3271

**Pull request merged**